gwswlib/hydx.py

```diff
@@ -8,6 +8,47 @@ class Hydx:
     def __init__(self):
         self.connection_nodes = []
 
+    def check_import_data(self):
+        self._check_on_unique(
+            self.connection_nodes, "identificatieknooppuntofverbinding", True, "knoop"
+        )
+
+    def _check_on_unique(
+        self,
+        records,
+        unique_field,
+        remove_doubles=False,
+        item_name_for_logging="",
+        log_level=logging.WARNING,
+    ):
+
```

Comment:
Docstring toevoegen met wat je teruggeeft.

Even oppassen trouwens. Qua naam "check" je wat, terwijl je in de return waarde zowel een check doet (true/false) als de dubbelingen teruggeeft.
Werkt het als je het "find_doubles()` noemt en dan checkt of je iets terugkrijgt? Dan splits je het checken en het vinden netjes.